docs/dashboards/data-modifiers.md

```diff
@@ -0,0 +1,80 @@
+Data Modifiers
+===
+
+The data comes in different shapes and sizes from all over the internet, so there needs to be a way of adjusting its format, so that it is accessible to the Dashboards. Instead of manually parsing the data, there is an option to use the `DataModifiers`. Those modifiers allow the user to modify the data on the initial load, as well as the end user to modify visible data through the GUI. Each of the modifiers is accessible through the main Dashboards package, except of the `MathModifier` which is most complex one, and requires loading a seperate module from `https://code.highcharts.com/dashboards/modules/math-modifier.js`
+
+### Where the modifier is defined?
+The modifier is attached to the connector that is defined in the [dataPool](https://www.highcharts.com/docs/dashboards/data-handling), in its options.
+
+### Types of modifiers
+The types of modifiers that are available are:
+
+* Chain (Allows to combine multiple modifiers in the same connector)
+* Range
+* Invert
+* Math
+
+### How to add the modifier?
+To add the modifier, you need to specify it in the options of the connector.
+```javascript
+connector: {
+    type: 'CSV',
+    id: 'connector-id',
+    options: {
+        //... other options
+        dataModifier: {
+            type: 'Math' // or 'Invert'|'Math' etc,
+            // additional modifier options
+        }
+    }
+}
+```
+The modifier is modifing the `table.modified` property of the connector, which leaves the original data unchanged, so that there is a way of restoring the original data if needed.
```

Comment:
```suggestion
The modifier modifies the `table.modified` property of the connector, which leaves the original data unchanged so that there is a way of restoring the original data if needed.
```